data/controller.properties

```diff
@@ -0,0 +1,4 @@
+#HSQL Database Engine 2.3.0
```

Comment:
I believe this is a generated file; we don't want it to be included in Git.
Maybe consider adding it to `.gitignore`.